apps/src/code-studio/utils.js

```diff
@@ -48,3 +48,17 @@ export function updateQueryParam(param, value) {
   }
   window.history.pushState(null, document.title, newLocation);
 }
+
+/**
+ * We indicate that a page is Rtl by setting the dir attribute on the root element
```

Comment:
The problem there, is I suspect we have lots of pages that depend on the DOM element being set (for CSS purposes) that don't load our JS. It might make sense for our setRtl action creator to make sure it updates the DOM status to be the same, but I don't think that can be the only place we set it.